### ISSUE TITLE
Implementing paths in Visualizing Data with Bokeh and Pandas

### ISSUE THREAD

**Issue by noramcgregor** [programminghistorian/jekyll#1662]
At British Library we frequently choose Programming Historian tutorials for our staff Hack & Yacks each month because they offer awesome and accessible intros to advanced digital topics. This was the latest a room of over 25 BL curators tackled--great tutorial and many thanks to the author! One place where things got muddled for us though was what follows the miniconda install. There are really two paths one can take at the "Running Code Examples" point, either proceed and do all the coding by hand or run the Jupyter notebook provided, but this isn't made very clear in the text and folks started writing .py files and such and then trying to open the Jupyter notebook. Just got a bit confused. For our group, the majority found the Jupyter notebook especially helpful as they're beginners and just wanted to get a sense of how the code all operates it, rather than write it from scratch. So just a suggestion that the lesson could benefit more learners by clearly offering these two different routes available to them. Cheers!

**Comment by drjwbaker** [comment]
Thanks @noramcgregor. I love that you use PH for your Hacks & Yacks: perhaps I can sit in on one in the future to get a sense of how people get on with PH lessons (we're always keep to learn more about our audience).

I spoke with @noramcgregor about this yesterday, and I can see the value of greater clarity around the [Running Code Examples](https://programminghistorian.org/en/lessons/visualizing-with-bokeh#running-code-examples) section so that readers know that one option for 'taking' the lesson is to not install the VE and instead just work through the Jupyter Notebook.

@svmelton: as English Managing Editor, how do you want to handle this? This lesson hasn't been translated and it doesn't look like it is in the lesson pipeline for either the ES or FR editions, so we have an opportunity to fix it now before it is. As the original editor, I think it is on me to work on this, even though I'm no longer on the @programminghistorian/english-team and I'm not 100% sure of your processes, so I'm happy to follow your lead!

**Comment by svmelton** [comment]
Hi @drjwbaker—I'm happy for you to take this on if you're up for it!

**Comment by drjwbaker** [comment]
@svmelton I'm a bit stretched (as per my email) but I can make a start.

@archaeocharlie: we have a comment on your lesson (see above). The ask is that there is greater clarity around the [Running Code Examples](https://programminghistorian.org/en/lessons/visualizing-with-bokeh#running-code-examples) section so that readers know that one option for 'taking' the lesson is to not install the VE and instead just work through the Jupyter Notebook.

Are you willing and able to take a look at this and offer some thoughts? If so, and you agree that a light edit might be in order, we can work together on implementation.

And congrats on publishing such a well thought of lesson!

**Comment by drjwbaker** [comment]
@svmelton On reflection, any chance that you or one of the EN team could ake this? I'm more over-committed than I realised...

**Comment by archaeocharlie** [comment]
Hi @svmelton @drjwbaker. I'm happy that it's getting use and will work on clarifying this section. I should have time to make changes at the end of next week.

**Comment by drjwbaker** [comment]
@archaeocharlie Fab. I really appreciate it. When you are ready, please raise a PR for the suggested changes and tag @svmelton and I. Though if you'd prefer to post the changes in this thread, that is good as well.

**Comment by walshbr** [comment]
@archaeocharlie I'm going through and checking on old tickets. Did you get a chance to look at this?

**Comment by mariajoafana** [comment]
Hi @walshbr shall we contact @archaeocharlie via email regarding this?

**Comment by walshbr** [comment]
@mariajoafana - @svmelton said on slack that she would reach out.

**Comment by svmelton** [comment]
I've dropped @archaeocharlie a note!

**Comment by rivaquiroga** [comment]
As discussed in https://github.com/programminghistorian/jekyll/issues/1941, I'll be supporting the bug assesment/fixing process. 
This one doesn't seem to be a bug in the terms defined in https://github.com/programminghistorian/jekyll/issues/1908#issuecomment-765692442, so I'll remove the label. It looks more like an issue with the clarity of the instructions given in the lesson.  
If the author is not responding, maybe one of the new editors can work on this. Could be a good opportunity to gain GitHub experience if they are not familiar with the workflow yet.

**Comment by anisa-hawes** [comment]
Hello @drjwbaker and @rivaquiroga. Based on the suggestion in the initial comment, I'd like to propose that we add an alert information box at the Running Code Examples step. 

I've drafted the following, in a Branch called [Issue-1662](https://github.com/programminghistorian/jekyll/commit/7440e78d21a59ca0ce04734708993df6f1e0f056). Let me know if you feel this sufficiently clarifies the options readers have at this stage of the lesson.

N.B. The alert boxes built with Bootstrap don't render in the GitHub preview, but I'm fairly confident that the formatting is correct.

One thing I noticed is that the link to the asset directs to the Submissions Repository: https://raw.githubusercontent.com/programminghistorian/ph-submissions/gh-pages/assets/visualizing-with-bokeh/visualizing-with-bokeh.ipynb Is that correct? Or should links to assets in published lessons direct to the assets in Jekyll? i.e., https://github.com/programminghistorian/jekyll/blob/gh-pages/assets/visualizing-with-bokeh/visualizing-with-bokeh.ipynb
